x-pack/plugins/ml/public/ui_actions/index.ts

```diff
@@ -14,6 +14,7 @@ import { createApplyEntityFieldFiltersAction } from './apply_entity_filters_acti
 import { createApplyInfluencerFiltersAction } from './apply_influencer_filters_action';
 import { createApplyTimeRangeSelectionAction } from './apply_time_range_action';
```

Comment:
I am now seeing the 'Apply time range' action fail, and this error in the console:

<img width="983" alt="Screenshot 2024-04-08 at 16 44 35" src="https://github.com/elastic/kibana/assets/7405507/382f002d-472c-4fff-8f16-8d6ae7c74ffb">

Reply:
Tested and confirmed this is fixed by https://github.com/elastic/kibana/pull/179547/commits/77c4b3db5756dd65f6519792d5c11bcb8f2e76a5